ext/standard/config.w32

```diff
@@ -1,6 +1,18 @@
 // vim:ft=javascript
 // $Id$
 
+ARG_WITH("argon2", "Argon2 support", "no");
+
+if (PHP_ARGON2 != "no") {
+	if (CHECK_LIB("Argon2Ref.lib", "argon2", PHP_ARGON2)
+	&& CHECK_HEADER_ADD_INCLUDE("argon2.h", "CFLAGS_ARGON2")) {
+		AC_DEFINE('HAVE_ARGON2LIB', 1);
+		EXTENSION("argon2", null, false);
```

Comment:
Hi! Thanks for pointing me in the right direction! I really appreciate it.

 It looks like the trick is to set the second arguement of `CHECK_LIB` (`target`) to `null` instead of `""`. An empty string creates `LIBS_` in the `Makefile`, whereas `null` correctly assigns it to `LIBS`.

Thanks again!